sites/fast-website/src/app/components/fast-frame/fast-frame.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-components";
+
+export const FastFrameStyles = css`
+    ${display("block")} :host {
+        flex-direction: column;
+        font-family: var(--body-font);
+        color: var(--neutral-foreground-rest);
+        box-sizing: border-box;
+        text-align: center;
+    }
+
+    .icon {
+        pointer-events: none;
+    }
+
+    .wrapper {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        height: 500px;
+    }
+
+    .content {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        align-items: start;
+    }
+
+    .preview {
+        display: flex;
+        background: #c8c8c8;
```

Comment:
Should this color be calculated?

Reply:
This is just temporary to show the bounds of the preview container.